src/main/java/org/swisspush/redisques/lua/LuaScriptState.java

```diff
@@ -93,6 +93,10 @@ public void loadLuaScript(final RedisCommand redisCommand, int executionCounter)
             } else {
                 log.info("load lua script for script type: " + luaScriptType);
                 redisClient.scriptLoad(script, stringAsyncResult -> {
+                    if (stringAsyncResult.failed()) {
+                        log.error("Received failed message for loadLuaScript.", stringAsyncResult.cause());
+                        return; // "https://softwareengineering.stackexchange.com/a/190535"
```

Comment:
If we can't upload LUA Script, then RedisQues will have heavy troubles.
Shouldn't we prevent to start the whole verticle instead?

Reply:
That’s a good question but it surly is not a topic of this PR. Please create a new issue.